### PR TITLE
Add support for showing and hiding the bid progress state

### PR DESCRIPTION
**Pull request by orta**
Relies on the VC receiving `LiveAuctionBiddingProgressState` from the socket at some point, should handle most of the work, only thing is triggering the buttons enable'd ness. Which can be another PR.

### CODE REVIEW

File: Artsy/Categories/Apple/UIView+Spinner.h [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#import <UIKit/UIKit.h>
+
+
+@interface UIView (Spinner)
```
Comment (orta) [inline Artsy/Categories/Apple/UIView+Spinner.h]: was in `ARSpinner` - now it's available to spin _all_ the views

Comment (ashfurrow) [inline Artsy/Categories/Apple/UIView+Spinner.h]: :party-parrot:

Comment (orta) [inline Artsy/Categories/Apple/UIView+Spinner.h]: hah - that worked in slack

File: Artsy/View_Controllers/Live_Auctions/LiveAuctionBidViewController.swift [hunk 2]
```diff
@@ -116,7 +115,7 @@ class LiveAuctionBidViewController: UIViewController {
     @IBOutlet weak var numberOfCurrentBidsLabel: UILabel!
     @IBOutlet weak var priceOfCurrentBidsLabel: UILabel!
 
-    func updateCurrentBidInformation(_: NSDate) {
+    private func updateCurrentBidInformation(_: NSDate) {
```
Comment (ashfurrow) [inline Artsy/View_Controllers/Live_Auctions/LiveAuctionBidViewController.swift]: You reckon its worth moving the `private` functions into an extension, for clarity?

File: Artsy/View_Controllers/Live_Auctions/LiveAuctionBidViewController.swift [hunk 3]
```diff
@@ -152,4 +151,40 @@ class LiveAuctionBidViewController: UIViewController {
         updateBiddingControls(bidViewModel.currentBid)
     }
 
+    // Incase you're new to storyboards, these views are on an associated
+    // view ( its on the top bar for the scene. ) which can be hidden
```
Comment (ashfurrow) [inline Artsy/View_Controllers/Live_Auctions/LiveAuctionBidViewController.swift]: Could be worth linking to https://blog.curtisherbert.com/secondary-views/ – but hopefully this'll become common enough practice not to warrant it.

File: Artsy/View_Controllers/Live_Auctions/Resources/LiveAuctions.storyboard [hunk 4]
```diff
@@ -37,21 +38,21 @@
                         <rect key="frame" x="0.0" y="0.0" width="600" height="600"/>
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
                         <subviews>
-                            <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" misplaced="YES" text="LOT 10" textAlignment="natural" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="uXQ-J5-tSz">
```
Comment (ashfurrow) [inline Artsy/View_Controllers/Live_Auctions/Resources/LiveAuctions.storyboard]: State and XML, my two least favourite things.

File: Artsy/Categories/Apple/UIView+Spinner.h (hunk 1 above)
Comment (ashfurrow) [inline Artsy/Categories/Apple/UIView+Spinner.h]: :8ball:

**Comment by ashfurrow** [comment]
Looks good to merge, none of those comments are blockers.

File: Artsy/View_Controllers/Live_Auctions/LiveAuctionBidViewController.swift (hunk 2 above)
Comment (orta) [inline Artsy/View_Controllers/Live_Auctions/LiveAuctionBidViewController.swift]: I thought of that at first, but I can't keep the IBOutlets next to their funcs then

**Pull request merged**